.github/workflows/build.yml

```diff
@@ -25,32 +27,37 @@ jobs:
 
     strategy:
       matrix:
-        os: [macos-latest, windows-latest]
+        os:
+          - macos-latest
+          - windows-latest
 
     steps:
       - name: Checkout Code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Setup Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 20
 
-      - name: Get Mac API Key
+      - name: Set Mac API Key
         if: startsWith(matrix.os, 'macos')
         run: |
           mkdir -p ~/private_keys/
           echo '${{ secrets.APPLE_API_KEY_CONTENT }}' > ~/private_keys/AuthKey_${{ secrets.APPLE_API_KEY_ID }}.p8
 
       - name: Install dependencies
-        run: cd app && npm install
+        working-directory: app
+        run: npm install
 
       - name: Set environment variables
+        working-directory: app
         run: |
-          echo "VITE_SUPABASE_API_URL=${{ secrets.SUPABASE_API_URL }}" >> app/.env
-          echo "VITE_SUPABASE_ANON_KEY=${{ secrets.SUPABASE_ANON_KEY }}" >> app/.env
-          echo "VITE_MIXPANEL_TOKEN=${{ secrets.MIXPANEL_TOKEN }}" >> app/.env
-          cat app/.env
```

Comment:
This was not a good idea.

Reply:
Oof good catch, must've been left over from testing. Will rotate keys

Reply:
Keys are not displayed because GHA never displays secrets.

Reply:
![image](https://github.com/user-attachments/assets/967ff42a-de67-4f27-89ae-3be818d6cc11)

Reply:
That's smart of them. Even better!

Reply:
Try `tac` 🙃